themes/_core/js/checkout-payment.js

```diff
@@ -109,12 +111,23 @@ class Payment {
     return `.js-payment-${moduleName}`;
   }
 
+  showNativeFormErrors () {
+    $(this.termsCheckboxSelector).get(0).reportValidity();
+    $('input[name=payment-option]').get(0).reportValidity();
+  }
+
   confirm() {
-    var option = this.getSelectedOption();
-    if (option) {
-      $(this.confirmationSelector + ' button').prop('disabled', true);
-      $('#pay-with-' + option + '-form form').submit();
+    let option = this.getSelectedOption();
+    let termsAccepted = this.haveTermsBeenAccepted();
+
+    if (option === undefined || termsAccepted === false) {
+      this.showNativeFormErrors();
+
+      return;
     }
+
+    $(this.confirmationSelector + ' button').addClass('disabled');
+    $('#pay-with-' + option + '-form form').submit();
```

Comment:
This is processed by webpack right ?

Would it be possible to do this at the same time ?

```suggestion
    $(`${this.confirmationSelector} button`).addClass('disabled');
    $(`#pay-with-${option}-form form`).submit();
```

Reply:
it's successfully processed by weback.
did so and also changed the other concatenations